jobbergate-api/jobbergate_api/apps/job_submissions/schemas.py

```diff
@@ -60,9 +60,218 @@
         description="The report message received from cluster-agent when a job submission is rejected",
         example="Unrecognized SBATCH arguments",
     ),
+    execution_parameters=MetaField(
+        description="The parameters to be passed to the job submission",
```

Comment:
Can you please include a link to the Slurm documentation for accepted parameters?

Reply:
solved in [5c44cbe](https://github.com/omnivector-solutions/jobbergate/pull/189/commits/5c44cbe56ba5551c97cd0c1d9ae7739980153270)